### PR TITLE
Simple code and typo fixed in pkg/kubelet

### PR DESCRIPTION
**Pull request by denkensk**
What this PR does / why we need it:
Simple code and typo fixed.
Which issue this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close that issue when PR gets merged): fixes #
none
Special notes for your reviewer:
none
Release note:
none

### CODE REVIEW

**Comment by denkensk** [comment]
@dims

**Review by dims — APPROVED** [review]

**Comment by denkensk** [comment]
@Random-Liu
@yujuhong 
thank you~~~

File: pkg/kubelet/reason_cache.go [hunk 1]
```diff
@@ -40,8 +40,8 @@ type ReasonCache struct {
 	cache *lru.Cache
 }
 
-// Reason is the cached item in ReasonCache
-type reasonItem struct {
+// ReasonItem is the cached item in ReasonCache
+type ReasonItem struct {
```
Comment (k82cn) [inline pkg/kubelet/reason_cache.go]: let keep this as private :)

**Comment by k82cn** [comment]
LGTM overall, as it's code clear :)

File: pkg/kubelet/reason_cache.go (hunk 1 above)
Comment (denkensk) [reply pkg/kubelet/reason_cache.go]: exported method Get returns unexported type *kubelet.reasonItem on line 97
```go
func (c *ReasonCache) Get(uid types.UID, name string) (*reasonItem, bool)
```
I think if I need to change reasonItem to exported?

**Pull request closed without merging**